requirements.txt

```diff
@@ -12,3 +12,5 @@ websockets==7.0
 appdirs==1.4.3
 multidict==4.4.2
 motor==2.0.0
+git+https://github.com/tinloaf/matrix-python-sdk@groups
```

Comment:
@Cadair is there a more stable position for this now that I can point to? Alternatively, @jacobtomlinson is there a way of specifying a repo as a requirement that doesn't break the tests?

Reply:
This should be my async api package now? We don't need to rely on the groups branch for this.

Reply:
Sadly we use the `requirements.txt` file in the `setup.py` too for dependencies, so a git style requirement is not supported. I recommend using (or publishing your own) package on pypi/conda-forge.

Reply:
this line should just be `matrix-api-async`